src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -29,9 +32,12 @@ public FindCommand parse(String args) throws ParseException {
         FindCommand findCommand;
 
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_ID);
-
-        if (arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ID) // user inputted both name and id
+                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_ID, PREFIX_MODULE_CODE);
+        // user inputted name, id, modcode or any two combinations of the two
```

Comment:
```suggestion
        // user inputted more than one search term (ie. name, id, or moduleCode)
```

Perhaps this might be make it a little clearer?